package.json

```diff
@@ -16,6 +19,17 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,json,css}": [
+      "prettier --single-quote --write",
```

Comment:
As per I know how `prettier` works, we are only setting a single rule `--single-quote`. It worth integrating prettier with eslint.